### PR TITLE
Record: val_bpb = 1.06128 SmearGate BOS Fix + PR #1787 Base + Smear Gate + LQER Asymmetric + Phased TTT (indirect 3 seed mean)

### PR DESCRIPTION
**Pull request by aquariouseworkman**
# Record: SmearGate BOS Fix + PR #1787 Base + Smear Gate + LQER Asymmetric + Phased TTT

**val_bpb = 1.06128** | **~15.95 MB** | 8xH100 SXM

## Result

| Seed | Pre-TTT BPB | Post-TTT BPB | Artifact (bytes) |
|------|-------------|--------------|------------------|
| 42   | 1.07406     | **1.06128**  | 15,952,086       |

Merged SOTA (PR #1493): **1.0810 BPP**. Delta: **-0.0197 BPP**. Clears the 0.005-nat threshold.
3-seed validation (seeds 42 / 314 / 1234) is provided in PR #1868, which evaluates this exact record package. The mean across those three seeds is reported there.

## Key Change: SmearGate BOS Document Boundary Fix

Builds on PR #1797 stack (PR #1787 base + SmearGate + LQER Asymmetric) but fixes the **SmearGate cross-document leakage bug** identified by @cocohearts in PR #1797 audit.

The bug: SmearGate 1-token causal lookback does not mask BOS positions, so the final token of document N smears into BOS of document N+1.

The fix (applied in both forward_logits and forward_ttt):

    bos_mask = (input_ids[:, 1:] == 1).unsqueeze(-1)
    g = g.masked_fill(bos_mask, 0.0)

## Technique Stack

| Component | Origin |
|-----------|--------|
| CaseOps bijective case transform | PR #1729 / PR #1736 |
| SparseAttnGate | PR #1787 (nprime06) |
| SmearGate + BOS fix | PR #1797 + this submission |
| LQER asymmetric rank-4 | PR #1797 |
| Phased TTT (score-first, 3 phases) | PR #1394 / PR #1736 |
| PolarNS + MIN_LR=0.1 + FusedCE | PR #1787 |
| Full Hessian GPTQ + Brotli | PR #1019 / PR #1530 |

## Architecture

11L x 512d x 8H/4KV, MLP 4x, LeakyReLU(0.5)^2, Partial RoPE (16/64 dims), layerwise LN scale, tied embeddings, logit softcap=30.0. Depth recurrence: layers 3-5 looped x2 (activated at frac=0.35). Parallel residuals from layer 8. XSA on all 11 layers. SmearGate window=12.

## Compliance

- Artifact <= 16,000,000 bytes: 15,952,086 bytes
- train_time <= 600s: 599.6s
- eval_time <= 600s: 519.5s
- Issue #1017 Conditions 1-4: All satisfied. SmearGate BOS mask ensures no cross-document leakage.

## Credits

- @nprime06 -- PR #1787 base stack
- @romeerp -- CaseOps transform (PR #1729)
- @dexhunter -- SmearGate + LQER (PR #1797)
- @cocohearts -- Identifying SmearGate BOS bug
- @abaybektursun -- Score-first TTT (PR #549)
- @clarkkev -- GPTQ SDClip + SP8192 (PR #1394)

### CODE REVIEW

**Comment by h1beee** [comment]
need results on 3 different seeds

**Comment by aquariouseworkman** [comment]
> need results on 3 different seeds

I ran out of credits on RunPod. This amazing person validated the other 2 for me! 

see: https://github.com/openai/parameter-golf/pull/1868

**Comment by aquariouseworkman** [comment]
Correct to last comment (edited) 
See for 3 mean test: https://github.com/openai/parameter-golf/pull/1868

**Review by cocohearts — CHANGES_REQUESTED** [review]
Accepted on substance for the original SmearGate BOS-fix submission, but this PR is not merge-ready in its current scope. It now contains a second, later AWQ-lite submission directory, `records/track_10min_16mb/2020-04-29_AWQ_lite_mixedprecision_GPTQ`, with an invalid 2020 date and a separate ML change that was not part of the accepted #1851 leaderboard row. Please split that newer AWQ-lite work into a separate PR or remove it from this one. This PR should merge only the accepted 2026-04-27 BOS-fix record package, with its 3-seed support either included directly or clearly tied to #1868.

**Review by cocohearts — DISMISSED** [review]
Accepted on substance for the original SmearGate BOS-fix submission, but this PR is not merge-ready in its current scope. It now contains a second, later AWQ-lite submission directory, `records/track_10min_16mb/2020-04-29_AWQ_lite_mixedprecision_GPTQ`, with an invalid 2020 date and a separate ML change that was not part of the accepted #1851 leaderboard row. Please split that newer AWQ-lite work into a separate PR or remove it from this one. This PR should merge only the accepted 2026-04-27 BOS-fix record package, with its 3-seed support either included directly or clearly tied to #1868.

**Comment by aquariouseworkman** [comment]
Fixing now

**Comment by aquariouseworkman** [comment]
> Accepted on substance for the original SmearGate BOS-fix submission, but this PR is not merge-ready in its current scope. It now contains a second, later AWQ-lite submission directory, `records/track_10min_16mb/2020-04-29_AWQ_lite_mixedprecision_GPTQ`, with an invalid 2020 date and a separate ML change that was not part of the accepted #1851 leaderboard row. Please split that newer AWQ-lite work into a separate PR or remove it from this one. This PR should merge only the accepted 2026-04-27 BOS-fix record package, with its 3-seed support either included directly or clearly tied to #1868.

Fixed, please verify
